plugins/search-elasticsearch-rest/src/main/java/com/thinkbiganalytics/search/ElasticSearchRestService.java

```diff
@@ -149,13 +148,18 @@ public void commit(@Nonnull String indexName) {
 
     @Override
     public void index(@Nonnull String indexName, @Nonnull String typeName, @Nonnull String id, @Nonnull Map<String, Object> fields) {
+        index(indexName, typeName, id, fields, null);
+    }
+
+    @Override
+    public void index(@Nonnull String indexName, @Nonnull String typeName, @Nonnull String id, @Nonnull Map<String, Object> fields, String parentId) {
         buildRestClient();
         try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)